lib/views/pay/numberpad_view.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:clnapp/utils/app_provider.dart';
+import 'package:flutter/material.dart';
+
+import '../../api/api.dart';
+import '../../components/buttons.dart';
+import '../../model/app_model/pay_invoice.dart';
```

Comment:
please use the package import

Reply:
I am so sorry, Everytime I forgot to fix this